### ISSUE TITLE
run-flake8 confused after rebase

### ISSUE THREAD

**Issue by citibeth** [LLNL/spack#1911]
OK, now I know I'm not on an old branch.  I just rebased my PR onto dev:

https://github.com/LLNL/spack/pull/1250

Now I get this when I run flake8 tests:
```
$ ./run-flake8-tests 
Dependencies found.
./run-flake8-tests: line 31: ./changed_files: No such file or directory
No Python files were modified.
```

:(

**Comment by citibeth** [comment]
@adamjstewart

**Comment by adamjstewart** [comment]
One step ahead of you again! I just noticed this and fixed it in #1883. Specifically, see https://github.com/LLNL/spack/pull/1883/files#r81472234

**Comment by adamjstewart** [comment]
Closing now that #1883 has been merged.
